omigami/models/estimator.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+from typing import Any
+from sklearn.base import BaseEstimator, clone
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import RandomForestClassifier
+from xgboost import XGBClassifier
+from omigami.data import RandomState
+from omigami.models.pls_classifier import PLSClassifier
+
+
+class ESTIMATORS:
+    RFC = "RFC"
+    XGBC = "XGBC"
+    PLSC = "PLSC"
+
+
+class Estimator:
+    def __init__(self, estimator: Any, random_state: RandomState):
```

Comment:
Maybe we shouldn't dictate the user how to intialize his Estimator class, maybe he want's do implement it directly so he wouldn't need estimator attribute.

Reply:
> so he wouldn't need estimator attribute.

what do you mean?